scripts/sysman/EXT_Exp_RasState.rst

```diff
@@ -0,0 +1,89 @@
+<%
+import re
+from templates import helper as th
+%><%
+    OneApi=tags['$OneApi']
+    s=tags['$s']
+    S=s.upper()
+%>
+:orphan:
+
+.. _ZES_extension_ras_state:
+
+========================
+ RAS Get State Extension
+========================
+
+API
+----
+
+* Functions
+
+    * ${s}RasGetStateExp
+
+* Enumerations
+
+    * ${s}_ras_state_exp_version_t
+    * ${s}_ras_error_category_exp_t
+
+* Structures
+
+    * ${s}_ras_state_exp_t
+
+
+RAS State
+~~~~~~~~~~~
+
+Reliability, Availability, Serviceability (RAS) is a standard mechanism to report HW errors.
+In L0 Sysman, we report these errors via RAS API and organized into Correctable or Uncorrectable
+errors. Furthermore, the errors can be grouped by category so that source of the error
+is easily identifiable.
+
+This extension is defined with intent to provide an extensible interface to the user for discovering
+these errors. A separate function is provided to allow users to clear the error counters.
+This functionality extends and is intended to eventually replace existing mechanism via ${s}RasGetState.
+
+To that end, this extension also adds new enumerator for RAS error categories. The list of error categories
+include existing ones (refer  Sysman Programming guide) and also additional ones which are defined here.
+The additional error categories are listed below: 
+
++-------------------------------------------------------------------+--------------------------------------------------------------+----------------------------------------------------------------+
+| Error category                                                    | ${S}_RAS_ERROR_TYPE_CORRECTABLE | ${S}_RAS_ERROR_TYPE_UNCORRECTABLE |
++===================================================================+==============================================================+================================================================+
+| ${S}_RAS_ERROR_CAT_MEMORY_ERRORS                                   | Number of ECC                                                | Number of ECC                                                  |
```

Comment:
This programming guide should use the new enum type values, `${S}_RAS_ERROR_CATEGORY_EXP_MEMORY_ERRORS`

Reply:
fixed